apps/settings/test/unit/panels/findmydevice/findmydevice_test.js

```diff
@@ -246,18 +274,14 @@ suite('Find My Device panel > ', function() {
   });
 
   test('wake up find my device upon a disable attempt', function() {
-    this.sinon.stub(window, 'wakeUpFindMyDevice');
     checkbox.checked = true;
-
     var event = new CustomEvent('click', {
       bubbles: true,
       cancelable: true
     });
     checkbox.querySelector('label').dispatchEvent(event);
-
-    assert.ok(window.wakeUpFindMyDevice.calledWith(
-        IAC_API_WAKEUP_REASON_TRY_DISABLE));
-    window.wakeUpFindMyDevice.reset();
```

Comment:
I had some trouble with setting up stub properly. It's now been fixed and I've added the reset() back.